code/mob/living/silicon/ai.dm

```diff
@@ -1393,6 +1427,33 @@ var/list/ai_emotions = list("Happy" = "ai_happy",\
 
 	src.show_laws(0)
 	return
```

Comment:
uslesss

Reply:
Sorry, having a hard time understanding what you mean by 'uslesss'. Probably 7 am brain but could you explain a little?

Reply:
okay ASSUMING you were referring to the return statements (zjd pointed that out), I'm gonna resolve these now, but pls yell at me if that wasn't the intent